test/runtests.jl

```diff
@@ -86,17 +86,26 @@ function get_logging_level(env_name::String, default)
     return log_level
 end
 
+function get_exclude_from_debug_logging()
+    min_levels = Dict{Symbol, Base.LogLevel}()
+    groups = get(ENV, "ExcludeFromDebugLogging", "")
+    return Dict(Symbol(x) => Logging.Error for x in split(groups, ","))
+end
```

Comment:
Let me know if you have an idea that is less hacky. In my mind this is the easiest way to do what I want to do: exclude noisy log groups while I debug one particular thing. We could do the inverse and have something that says, "enable debug for only group X."

Reply:
If I understand this correctly, it only enables error messages for all groups defined in the "ExcludeFromDebugLogging" environment variable. I tend to like the paradigm where we as for a template that is pre-populated with defaults and then allow the user to modify. So we could do something like:
```
min_levels = get_log_levels(logger)
set_min_level!(logger, :TimeSeries, Logging.Debug)
```

Reply:
My concern is the user knowing what groups are available

Reply:
If it is an environment variable, I think it should be `SIIP_EXCLUDE_FROM_DEBUG_LOG`, i.e. application name followed by underscore followed by variable name, all upper case.

Reply:
Ok, now I see `get_min_levels` should serve this purpose.

Reply:
Agree with @kdheepak that it should be UPPER_CASE if it is an environment variable.

@claytonpbarrows You can do this:
```
ENV["SIIP_EXCLUDE_FROM_DEBUG_LOG"] = "TimeSeries,Serialization,System"
```

I started with an environment variable because that's what we already use for enabling debug logging. A config file may be a better solution. Imagine
```
$ cat log_config.toml
SIIP_CONSOLE_LEVEL = Info
SIIP_FILE_LEVEL = Debug
[Group Levels]
System = Debug
Serialization = Info
TimeSeries = Info
```
Before running tests:
```
ENV["SIIP_LOG_CONFIGURATION"] = "log_config.toml"
```
We could do something similar when running PowerSimulations. If we wanted to be able to change dynamically then we could even have a file watcher detect changes.